components/script/dom/htmlbodyelement.rs

```diff
@@ -23,9 +25,8 @@ use crate::dom::htmlelement::HTMLElement;
 use crate::dom::node::{document_from_node, window_from_node, BindContext, Node};
 use crate::dom::virtualmethods::VirtualMethods;
 
-/// How long we should wait before performing the initial reflow after `<body>` is parsed, in
-/// nanoseconds.
-const INITIAL_REFLOW_DELAY: u64 = 200_000_000;
+/// How long we should wait before performing the initial reflow after `<body>` is parsed.
+const INITIAL_REFLOW_DELAY: Duration = Duration::from_nanos(200_000_000);
```

Comment:
Can we change this to `Duration::from_millis(200)` to make it even more explicit?

Reply:
Sure. I've done that now.